.github/workflows/docker-runtime-draft.yml

```diff
@@ -30,38 +53,77 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          ref: ${{ github.event.inputs.to }}
+          ref: ${{ steps.check-git-ref.outputs.git_ref }}
       - name: Setup Rust toolchain
         run: rustup show
-      - name: Build Node
-        run: cargo build --profile=production --all
-    
-      - name: Save parachain binary
+      - name: Build Node tanssi-relay
+        if:  contains(needs.set-tags.outputs.git_branch, 'starlight')
+        run: cargo build -p tanssi-relay --profile=production
+      - name: Build Node parachains
+        if:  contains(needs.set-tags.outputs.git_branch, 'para')
+        run: cargo build -p tanssi-node -p container-chain-frontier-node -p container-chain-simple-node --profile=production
+      - name: Save binary
         if: ${{ matrix.cpu == '' }}
         run: |
           mkdir -p binaries
-          cp target/production/tanssi-node binaries/tanssi-node
-          cp target/production/container-chain-frontier-node binaries/container-chain-frontier-node
-          cp target/production/container-chain-simple-node binaries/container-chain-simple-node
-      - name: Save parachain custom binary
+          if [[ "${{ needs.set-tags.outputs.git_branch }}" == *"para"* ]]; then
+            cp target/production/tanssi-node binaries/tanssi-node
```

Comment:
This can be combined to something like:
```
        suffix=""
        if [ "${{ inputs.cpu }}" != "" ]; then
          suffix="-${{ inputs.cpu }}"
        fi
        cp target/${{ inputs.profile }}/tanssi-node binaries/tanssi-node$suffix
```